README.md

```diff
@@ -33,10 +33,6 @@ Tokio project, but does _not_ require the `tokio` runtime to be used.
 
 ## Usage
 
-(The examples below are borrowed from the `log` crate's yak-shaving
-[example](https://docs.rs/log/0.4.10/log/index.html#examples), modified to
-idiomatic `tracing`.)
```

Comment:
The intention behind this note was less about giving credit to the `log` crate (it's MIT-licensed, too, so that's not necessary), and more to highlight to users familiar with `log` that these examples can be compared with `log`'s to see differences between idiomatic use of `tracing` and log.

I agree that it probably shouldn't be the first thing you see when you read these docs. Do you think that's valuable to have someplace, still maybe later in the docs?

Reply:
I looked at the linked examples in `log` and the code all seemed to be quite different. There's an `info!()` but even there it's not clear if the macro syntax corresponds exactly?

So, I think it's a nice in-joke that instrumentation examples shave yaks, but it's probably not relevant this readme?

Reply:
The examples may have also mutated enough since they were originally added that the comment may not really be valuable any longer --- I'm fine with removing it if it's no longer helpful.